src/components/ReanimatedSwipeable.tsx

```diff
@@ -544,27 +580,37 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       swipeableMethods,
     ]);
 
-    const animatedStyle = useAnimatedStyle(() => ({
-      transform: [{ translateX: appliedTranslation.value }],
-      pointerEvents: rowState.value === 0 ? 'auto' : 'box-only',
-    }));
+    const animatedStyle = useAnimatedStyle(
+      () => ({
+        transform: [{ translateX: appliedTranslation.value }],
+        pointerEvents: rowState.value === 0 ? 'auto' : 'box-only',
+      }),
+      [appliedTranslation, rowState]
+    );
+
+    const containerStyle = props.containerStyle;
+    const childrenContainerStyle = props.childrenContainerStyle;
+    const remainingProps = { ...props };
 
-    const composedGesture = Gesture.Race(panGesture, tapGesture);
     return (
-      <Animated.View
-        onLayout={onRowLayout}
-        style={[styles.container, props.containerStyle]}>
-        {leftElement}
-        {rightElement}
-        <GestureDetector
-          gesture={composedGesture}
-          touchAction="pan-y"
-          {...props}>
-          <Animated.View style={[animatedStyle, props.childrenContainerStyle]}>
-            {children}
-          </Animated.View>
-        </GestureDetector>
-      </Animated.View>
+      <GestureDetector gesture={panGesture} touchAction="pan-y">
+        <Animated.View
+          onLayout={onRowLayout}
+          style={[styles.container, containerStyle]}>
+          {leftElement}
+          {rightElement}
+          <GestureDetector gesture={tapGesture} {...remainingProps}>
```

Comment:
```suggestion
          <GestureDetector gesture={tapGesture}>
```

That's not necessary, it basically wouldn't do anything since gestures are configured in the builder in the new API.